samples/bluetooth/iso_time_sync/README.rst

```diff
@@ -0,0 +1,364 @@
+.. _bluetooth-isochronous-time-synchronization:
+
+Bluetooth: ISO time synchronization
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The Bluetooth® ISO time synchronization sample showcases time-synchronized processing of isochronous data.
+
+The sample demonstrates:
+
+ * How to send isochronous data on multiple streams in such a way that all receiving devices can process the data at the same time.
+ * How receiver(s) of the isochronous data can process received data synchronously.
+ * How to provide the isochronous data to the controller right before it is sent on air.
+ * How to achieve this for either broadcast (BIS) or over connections (CIS).
+
+This sample demonstrates this by configuring a single device as a transmitter of its **BTN1** state.
```

Comment:
What exactly is BTN1? We use ** just for items on the PCB (like gpio pins, or leds)

Reply:
It is button 1. I will change it to BUTTON1 instead